discord-roles.md

```diff
@@ -0,0 +1,71 @@
+| Helper                 | Experts in the community          | Given to experts.                                                 |        :x:         |        None         |                None                |
+| Snapshot Alarm         | Added via Discord Join            | Available for anyone to add in Channels & Roles.                  |        :x:         |        None         |                None                |
+| Announcement Pings     | Added via Discord Join            | Available for anyone to add in Channels & Roles.                  |        :x:         |        None         |                None                |
+| Dev Announcement Pings | Added via Discord Join            | Available for anyone to add in Channels & Roles.                  |        :x:         |        None         |                None                |
+| Snapshot Guesser       | Whoever guessed the snapshot time | Assigned by a team member.                                        |        :x:         |        None         |                None                |
+| All Vanity Roles       | All team members (optionally)     | The only roles with colors.                                       |        :x:         | 1 for each MC Color | 1 for each hoisted role + combined |
+| Nitro Booster          | Nitro boosters                    | Handled by the Discord integration.                               |        :x:         |        None         |         Nitro Booster Icon         |
+| All Bot Roles          | All bots                          | Discord-demanded integrations.                                    |        :x:         |        None         |                Gear                |
+
+### PIM
+PIM stands for Privileged Identity Management, its main goal is to provide a Just-In-Time way of giving users permissions when they need it.
+In a controlled and operational fashion. With or without approval of a second person.
+This ensures that roles with extreme rights, which when misused (either through hacking or for personal gain) would harm the server and the community.
+If an approval is needed for the role, it can only be granted by somebody other than the requester, even if the requester would normally be eligible to approve the role request.
+All team members will be eligible for any and all roles exposed via the bot.
+
+#### The PIM Bot
+The pim bot is a special sole purpose bot that can be used by team members to request certain roles.
+Its command structure is: `/pim <rolename> <reason>` depending on the roles configuration within the bot, the bot will them either directly assign your requested role to you, or ask a specific team for approval.
+When the allotted time for your assignment has passed, the bot will automatically remove the role from you. This way you do not forget, and the risk to the community is minimal.
```

Comment:
I've come to disagree on the idea of using a technical solution for what is a server management problem. Partly because it means we are directly beholden to that bot's functioning for managing the server, and without it we would be effectively paralyzed until we temporarily or permanently fix the problem.

I could perhaps agree if this bot supplements a permanent cadre of server administrators, as a fallback for when all those admins are offline and a situation is in dire need of correction--possibly implemented by temporarily creating the role and assigning the person to that role, and then deleting the role afterward--but handling all permissions control to a bot has risks and assumptions which I don't think outweigh the benefits.

Reply:
Added the admin account.